scaleway/resource_rdb_instance.go

```diff
@@ -186,12 +192,6 @@ func resourceScalewayRdbInstance() *schema.Resource {
 					},
 				},
 			},
-			"disable_public_endpoint": {
```

Comment:
Could you add examples of configuration in the documentation? I guess that examples need to be updated to showcase